resolver/resolver-https.go

```diff
@@ -94,6 +94,10 @@ func (hr *HTTPSResolver) Query(ctx context.Context, q *Query) (*RRCache, error)
 		_ = resp.Body.Close()
 	}()
 
+	if resp.StatusCode != http.StatusOK {
+		return nil, fmt.Errorf("http response code %s", resp.Status)
```

Comment:
Please be more descriptive and use a verb-form for the error. Eg: `http request failed with %s`

Reply:
```suggestion
		return nil, fmt.Errorf("http request failed with %s", resp.Status)
```